skimage/exposure/exposure.py

```diff
@@ -463,3 +463,28 @@ def adjust_sigmoid(image, cutoff=0.5, gain=10, inv=False):
 
     out = (1 / (1 + np.exp(gain * (cutoff - image / scale)))) * scale
     return dtype(out)
+
+
+def is_low_contrast(image, threshold_percent=0.5, lower_bound=1,
+                    upper_bound=99, method='linear'):
+    """Detemine if an image is low contrast.
+
+    Parameters
+    ----------
+    image : array-like
+        The image under test.
+    threshold_percent : float, optional
+        The low contrast threshold in percent.
+    lower_bound : float, optional
+        The lower bound percentile of the image to use.
+    upper_bound : float, optional
+        The upper bound percentile of the image to use.
+    method : str, optional
+        The contrast determination method.  Right now the only available
+        option is "linear".
+    """
+    dlimits = dtype_limits(image)
+    limits = np.percentile(image, [lower_bound, upper_bound])
+    ratio = (limits[1] - limits[0]) / float(dlimits[1] - dlimits[0])
```

Comment:
This might be controversial but I would write `from __future__ import division` at the top of the file and then forget about the float cast. Let's move everyone to the Python 3 way of life...